src/modules/commander/Commander.cpp

```diff
@@ -201,6 +201,7 @@ void print_status();
 bool shutdown_if_allowed();
 
 transition_result_t arm_disarm(bool arm, orb_advert_t *mavlink_log_pub, const char *armedBy);
+transition_result_t arm_disarm(bool arm, bool run_preflight_checks, orb_advert_t *mavlink_log_pub, const char *armedBy);
```

Comment:
Can we kill off the old function declaration?

Reply:
Done, that's a rebase fail.